packages/block-library/src/columns/editor.scss

```diff
@@ -148,26 +148,26 @@ div.block-core-columns.is-vertically-aligned-bottom {
 	right: 0;
 }
 
+// The empty state of a columns block has the default appenders.
+// Since those appenders are not blocks, the parent, actual block, appears "hovered" when hovering the appenders.
+// Because the column shouldn't be hovered as part of this temporary passthrough, we unset the hover style.
+.wp-block-columns [data-type="core/column"].is-hovered {
+	> .block-editor-block-list__block-edit::before {
+		content: none;
+	}
+
+	.block-editor-block-list__breadcrumb {
+		display: none;
+	}
+}
+
 // In absence of making the individual columns resizable, we prevent them from being clickable.
 // This makes them less fiddly. @todo: This should be revisited as the interface is refined.
 .wp-block-columns [data-type="core/column"] {
 	pointer-events: none;
 
-	// The empty state of a columns block has the default appenders.
-	// Since those appenders are not blocks, the parent, actual block, appears "hovered" when hovering the appenders.
-	// Because the column shouldn't be hovered as part of this temporary passthrough, we unset the hover style.
-	&.is-hovered {
-		> .block-editor-block-list__block-edit::before {
-			content: none;
-		}
-
-		.block-editor-block-list__breadcrumb {
-			display: none;
-		}
+	// This selector re-enables clicking on any child of a column block.
+	.block-core-columns .block-editor-block-list__layout {
+		pointer-events: all;
 	}
 }
-
-// This selector re-enables clicking on any child of a column block.
-:not(.components-disabled) > .wp-block-columns > .editor-inner-blocks > .editor-block-list__layout > [data-type="core/column"] > .editor-block-list__block-edit > * {
```

Comment:
Thanks so much for this reminder. I did forget to test reusable blocks. Thankfully this seems to work also as intended. GIF:

![gif](https://user-images.githubusercontent.com/1204802/55856653-388e7800-5b6b-11e9-93b1-231e47e8f196.gif)

Here's how the code works:

```
.wp-block-columns [data-type="core/column"] {
	pointer-events: none;

	// This selector re-enables clicking on any child of a column block.
	.block-core-columns .block-editor-block-list__layout {
		pointer-events: all;
	}
}
```

Basically the column block, and every child below, is made unclickable.

After that, only the `.block-editor-block-list__layout` element that is a child, is made clickable again, which appears to be sufficient.

The specificity is lower than that of any of the `.components-disabled` blocks, such as archives or reusable blocks, therefore the pointer-events: none attached to those is unaffected.

In the previous version, both > and * were used to make elements clickable again, which was both unnecessary because of inheritance, but also too specific, needing the :not selector.